conf/modules.config

```diff
@@ -35,3 +35,9 @@ process {
     }
 
 }
+
+if (params.platform == 'illumina') {
+    includeConfig 'modules_illumina.config'
+} else if (params.platform == 'nanopore') {
+    includeConfig 'modules_nanopore.config'
```

Comment:
is this not part of the repo? if so, i'd remove it

Reply:
like the separation tho

Reply:
> I am just submitting comments here since this is already merged. Let's wait for at least one approval in the future.

yes my bad, I got too trigger happy seeing Taylor's comment and thought it was an approval, I need to make that protection update to the repo to require approval

Reply:
what are you referring to for what isn't part of the repo?